src/domain/phone-code/index.types.d.ts

```diff
@@ -0,0 +1,11 @@
+interface IPhoneCodesRepository {
+  findRecent: (arg: {
```

Comment:
`find` prefix implied it returns an entity (that is the pattern we've been using so far). Since its returning a boolean perhaps a better name would be `checkExistance` or something like that.